doc/source/formats/topology_parsers.txt

```diff
@@ -3,30 +3,30 @@
 
 .. table:: Table of supported topology parsers and the attributes read
 
-    ======================================  ====================================================================  ===================================================================================================================================================================================================================  ====================
```

Comment:
Should this *.txt file have been ignored?

Reply:
I think I may have pushed changes before adding it to `.gitignore`.